modules/javafx.base/src/test/java/test/com/sun/javafx/collections/MappingChangeTest.java

```diff
@@ -49,7 +49,7 @@ public void setUp() {
 
     @Test
     public void testAddRemove() {
-        Change<Integer> change = new NonIterableChange.SimpleRemovedChange<Integer>(0, 1, new Integer(5), originalList);
+        Change<Integer> change = new NonIterableChange.SimpleRemovedChange<Integer>(0, 1, Integer.valueOf(5), originalList);
```

Comment:
Autobox?

Reply:
First two parameters are primitive type integer and the third parameter is template type.
Changing it to autobox would make it less readable.